depthai_nodes/ml/parsers/classification.py

```diff
@@ -31,20 +31,61 @@ class ClassificationParser(dai.node.ThreadedHostNode):
          An object with attributes `classes` and `scores`. `classes` is a list of classes, sorted in descending order of scores. `scores` is a list of corresponding scores.
     """
 
-    def __init__(self, classes: List[str] = None, is_softmax: bool = True):
+    def __init__(
+        self, archive_heads: list, head_name: str = "", is_softmax: bool = True
+    ):
         """Initializes the ClassificationParser node.
 
-        @param classes: List of class names to be used for linking with their respective
-            scores.
-        @param is_softmax: If False, the scores are converted to probabilities using
-            softmax function.
+        Attributes
+        ----------
+        archive_metadata : list
```

Comment:
Yes docstrings will be updated when we have decided on final names/adjustments

Reply:
I'd maybe also rename this parameter because its not so intuitive to me what `archive_heads`/`archive_metadata` stands for. Maybe naming it simply as `heads` or `head_metadata` would work?

Reply:
I see. Then I'd maybe rename this parameter because its not so intuitive to me what `archive_heads`/`archive_metadata` stands for. Maybe naming it simply as `heads` or `head_metadata` would work?